test/evp_test.c

```diff
@@ -1871,13 +1871,14 @@ static const EVP_TEST_METHOD encode_test_method = {
     encode_test_run,
 };
 
+
```

Comment:
useless change.

Reply:
The extra blank line was to match the extra spacing used elsewhere, such as before BASE64 TESTS.